src/envoy/http/authn/authenticator_base.cc

```diff
@@ -61,6 +61,32 @@ void AuthenticatorBase::validateX509(
   done_callback(&payload, true);
 }
 
+void AuthenticatorBase::validateTls(
+    const iaapi::MutualTls&,
+    const AuthenticatorBase::MethodDoneCallback& done_callback) const {
+  // In TLS connection, a client certificate may not always be present.
+  // If the client certificate is present, extract its identity.
+  ENVOY_LOG(debug, "AuthenticatorBase: {} this connection allows TLS",
```

Comment:
Please reduce this kind of debug message, we have too many. Delete old one when you come across as well.

Reply:
Done.